project_management/site_completeness_checklist copy.md

```diff
@@ -155,3 +159,8 @@ galleries such as:
 mediaqueri.es
 http://www.mobileawesomeness.com/
 http://creattica.com/
+
+
+[Rollbar]: https://rollbar.com
+[NewRelic]: https://newrelic.com
+[LogRotate]: https://support.rackspace.com/how-to/understanding-logrotate-utility/

```

Comment:
I'd never heard of log rolling until now. The fact that you're linking to an introductory article here is really helpful. It's enabling somebody like me to jump in and actually do this and set it up.

But for Rollbar/NewRelic, I don't really know what those are for, or why I'm supposed to use them. I wouldn't be able to go in and set either one up, because I wouldn't know what it's supposed to look like when I'm done. I wonder if there are introductory articles that would help people get up and running with these services?